spotify_player/src/cli/commands.rs

```diff
@@ -123,6 +123,16 @@ pub fn init_playback_subcommand() -> Command {
         )
 }
 
+pub fn init_search_command() -> Command {
+    Command::new("search").about("Search spotify").arg(
+        Arg::new("query")
+            .long("query")
+            .short('q')
```

Comment:
```suggestion
```

doesn't need to be an option, e.g it's better to call `search "query"` than `search --query "query"`